### ISSUE TITLE
Invalid parent directory when launch Tensorboard

### ISSUE THREAD

**Issue by zxsimple** [polyaxon/polyaxon#1204]
### Describe the bug
Tensorboard can't display chart after lanuching Tensorboard`polyaxon run --hub tensorboard:single-run -P uuid= 75e805dad60940519397a3dde42afcf9 -w`
`polyaxon-streams` pod reports error: `Invalid parent directory '/artifactsStore/aa8b1dbd1d674b08ad54042fd5da7a14'`
### To reproduce
1. Run the tutorial experiment by `polyaxon run -f quick-start/simple.yaml -l` and get the run uuid `75e805dad60940519397a3dde42afcf9 `
2. The artifacts can generate the event logs as `outputs\tensorboard\train\*` and `outputs\tensorboard\validate\*`
3. Launch tensorboard with command `olyaxon run --hub tensorboard:single-run -P uuid= 75e805dad60940519397a3dde42afcf9 -w`

### Expected behavior
Tensorboard chart can display the training graph.
### Environment
```
postgresql:
  persistence:
    enabled: true
    size: 5Gi
dns:
  backend: "coredns"
intervals:
  compatibilityCheck: -1
```

**Comment by polyaxon-team** [comment]
Are you running Polyaxon using a single node or multi-nodes?

**Comment by zxsimple** [comment]
Multi nodes on Kubernetes cluster. Just now I added PVC `artifactsStore` still can't find the path
```
artifactsStore:
  name: artifacts-store
  kind: volume_claim
  schema:
    mountPath: "/artifacts-store"
    volumeClaim: "polyaxon-artifacts-pvc"
```

**Comment by polyaxon-team** [comment]
Do you see the  artifacts in the artifacts tab in the UI?
Also, is the `polyaxon-artifacts-pvc` a multi-read-write volume?

**Comment by zxsimple** [comment]
I could see on the Experiment Job `artifacts` tab, but couldn't see on the Tensorflow service's tab.
> Also, is the polyaxon-artifacts-pvc a multi-read-write volume?

Yes, its `ReadWriteMany`.
```
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  annotations:
    volume.beta.kubernetes.io/storage-provisioner: everest-csi-provisioner
  name: polyaxon-artifacts-pvc
  namespace: polyaxon
spec:
  accessModes:
  - ReadWriteMany
  resources:
    requests:
      storage: 20Gi
  storageClassName: csi-nas
  volumeName: polyaxon-artifacts-pv
```

**Comment by polyaxon-team** [comment]
Can you show the logs of all containers in the tensorboard operation:  `polyaxon ops logs -a -uid TENSORBOARD_UUID`

**Comment by zxsimple** [comment]
Am I running the correct command? `polyaxon ops logs -a -uid 3f256e8753d34960a189c412a7e159f0`

```
2021-01-11 16:49:09.258055+08:00 | 192.168.1.19 | plx-operation-3f256e8753d34960a189c412a7e159f0-5689d86dcb-pnhlq | polyaxon-main | TensorBoard 2.2.1 at http://0.0.0.0:6006/services/v1/polyaxon/default/YellowRiverr/runs/3f256e8753d34960a189c412a7e159f0/ (Press CTRL+C to quit)
running
```

**Comment by polyaxon-team** [comment]
It should show the logs from all containers, it should show logs from init and sidecar containers as well.

**Comment by zxsimple** [comment]
Both of the commands has no output.
```
kubectl logs plx-operation-3f256e8753d34960a189c412a7e159f0-5689d86dcb-pnhlq -c polyaxon-init-artifacts-default -n polyaxon
kubectl logs plx-operation-3f256e8753d34960a189c412a7e159f0-5689d86dcb-pnhlq -c polyaxon-init-artifacts-artifacts-store -n polyaxon
```

**Comment by polyaxon-team** [comment]
That's strange. What version of Polyaxon are you running?

**Comment by zxsimple** [comment]
My polyaxon version is 1.4.4.

**Comment by polyaxon-team** [comment]
Something is not working as expected, feel free to join slack if you need help debugging the deployment, https://polyaxon.com/slack, to avoid spamming other users watching the repo.

**Comment by zxsimple** [comment]
OK

**Comment by polyaxon-team** [comment]
This regression was fixed. We patched the v1.5.0 containers, and we will release a new patch release v1.5.1 today.

Thank you for reporting this issue and for helping with debugging.

**Comment by zxsimple** [comment]
Great!
